MdePkg/Include/Library/DebugLib.h

```diff
@@ -4,10 +4,24 @@
   The Debug library supports debug print and asserts based on a combination of macros and code.
   The debug library can be turned on and off so that the debug code does not increase the size of an image.
 
-  Note that a reserved macro named MDEPKG_NDEBUG is introduced for the intention
+  Note that a reserved macro named MDEPKG_NDEBUG is introduced with the intention
   of size reduction when compiler optimization is disabled. If MDEPKG_NDEBUG is
   defined, then debug and assert related macros wrapped by it are the NULL implementations.
 
+  Current NULL implementations of these macros rely on the fact that directly unreachable

```

Comment:
I think the use of the word "Current" here is a bit confusing.
Maybe replace with "The"?